dj/models/materialization.py

```diff
@@ -29,10 +41,198 @@ class DruidMaterializationInput(GenericMaterializationInput):
     druid_spec: Dict
 
 
-class MaterializationOutput(BaseModel):
+class MaterializationInfo(BaseModel):
     """
     The output when calling the query service's materialization
     API endpoint for a cube node.
     """
 
+    output_tables: List[str]
     urls: List[AnyHttpUrl]
+
+
+class MaterializationConfigOutput(SQLModel):
+    """
+    Output for materialization config.
+    """
+
+    name: Optional[str]
+    engine: EngineInfo
+    config: Dict
+    schedule: str
+    job: str
+
+
+class MaterializationConfigInfoUnified(
```

Comment:
nit: This class name was a bit scary when I saw it further up but looking at the definition it makes sense. Is there a more intuitive naming we can use here? I'm thinking `s/MaterializationInfo/MaterializationsReported` and `s/MaterializationConfigInfoUnified/MaterializationInfo`? My logic is that the first model represents which materializations are reported by the query service (which tables) and the second model represents all info we have for a materialization (it's config as well as any reported materializations).

Reply:
Good call on the naming! Will update once my aircraft gets wifi 🙃 I'll also go through this PR again and see if I can correct confusing bits of naming.